src/implementationGuides/IGCompiler.ts

```diff
@@ -79,7 +79,7 @@ export class IGCompiler {
         const index: any = await loadJson(indexJson);
         const resources = [];
         for (const file of index.files) {
-            if (file.resourceType === resourceType) {
+            if (resourceTypes.includes(file.resourceType)) {
```

Comment:
Would this evaluate to true if `resourceTypes = ['SearchParameter', 'StructureDefinition']` and `resourceType = ['SearchParameter', 'StructureDefinition']` ?

Reply:
It wouldn't, but that scenario cannot happen.
`file.resourceType` is always a string, not an array of strings.